Avoid exporting types from core-auth that belong elsewhere (#8410)
Azure/azure-sdk-for-js: sdk/core/core-auth/CHANGELOG.md

# Release History

## 1.1.2 (Unreleased)

> - Remove the below interfaces from the public API surface as they have their source of truth defined elsewhere.

Reviewer: ```suggestion
- Remove the below interfaces from the public API of this package as they are defined elsewhere.
```
